.tina/queries/queries.gql

```diff
@@ -10,4 +10,17 @@ query consultingContentQuery($relativePath: String!) {
   consulting(relativePath: $relativePath) {
     ...ConsultingParts
   }
-}
\ No newline at end of file
+}
+
+query allTestimonialsQuery {
+  testimonialsConnection {
+    edges {
```

Comment:
can we rename the results as this litters the code with edge/node references which is very abstract

Reply:
Done - I gave names nodes and edges